core/src/test/java/org/mskcc/oncotree/utils/OncotreeTestConfig.java

```diff
@@ -131,6 +136,12 @@ public CrosswalkRepository crosswalkRepository() {
         return repository;
     }
 
+    @Bean
+    public SlackUtil slackUtil() {
+        SlackUtil slackUtil = Mockito.mock(SlackUtil.class);
+        return slackUtil;
```

Comment:
you return `Mockito.mock(SlackUtil.class)` if there isn't any customization going on